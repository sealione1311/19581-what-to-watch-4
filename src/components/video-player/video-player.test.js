import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

const src = `https://upload.wikimedia.org/wikipedia/commons/b/bb/2020-06-19_%E2%80%94_Fechner_monument%2C_Diepenheim.webm`;
const poster = `img/the-grand-budapest-hotel.jpg`;

it(`VideoPlayer should render correctly`, () => {
  const tree = renderer
      .create(
          <VideoPlayer
            src={src}
            poster={poster}
            isPlaying={false}
          />, {
            createNodeMock: () => {
              return {};
            }
          }).toJSON();

  expect(tree).toMatchSnapshot();
});
