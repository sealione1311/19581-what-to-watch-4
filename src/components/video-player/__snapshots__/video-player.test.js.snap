// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer should render correctly 1`] = `
<video
  className="video-player"
  height="175"
  muted="muted"
  poster="img/the-grand-budapest-hotel.jpg"
  width="280"
/>
`;
